modules/openxBidAdapter.js

```diff
@@ -431,9 +431,9 @@ function createVideoBidResponses(response, {bid, startTime}) {
     // true is net, false is gross
     bidResponse.netRevenue = true;
     bidResponse.currency = response.currency;
-    bidResponse.cpm = Number(response.pub_rev) / 1000;
-    bidResponse.width = response.width;
-    bidResponse.height = response.height;
+    bidResponse.cpm = parseInt(response.pub_rev, 10) / 1000;
```

Comment:
what about NaN ?

Reply:
the check here: https://github.com/openx/Prebid.js/pull/25/files#diff-2051ef6e55abda4fed56905b5982884aR425
should filter out the NaN case for pub_rev, but we can have it for width/height.  I felt it was too successive to check every field in the response for empty strings and undefined values